addons/account/views/company_view.xml

```diff
@@ -10,8 +10,49 @@
                     <field name="anglo_saxon_accounting"/>
                 </field>
                 <xpath expr="//group[@name='account_grp']" position="inside">
-                    <field name="bank_account_code_char" groups="base.group_no_one"/>
-                    <field name="accounts_code_digits" groups="base.group_no_one"/>
+                    <field name="bank_account_code_char"/>
+                    <field name="accounts_code_digits"/>
+                </xpath>
+                <xpath expr="//page[@name='general_informations']" position="inside">
```

Comment:
Remove this "Bank Accounts" section from the view. No need to have a one2manyn of bank accounts in the company.
Users will use the Journals view to configure journals, not going in the company form.